### ISSUE TITLE
Improve keyboard accessiblity

### ISSUE THREAD

**Issue by CodingSpiderFox** [elementary/files#2297]
### What Happened?

Kinda related to: https://github.com/elementary/onboarding/issues/217

When I have no working mouse, it is almost impossible the achieve simple file-related tasks with elementary files. 
It is too hard to see which I element in the files app is currently selected.

Especially the default folders like "Pictures", "Downloads", "Music", etc. in the left sidebar, as well as the mass storage devices (root file system, USB drives, network attached storages) are almost impossible to access using the tab / arrow keys on the keyboard

### Steps to Reproduce

Try to use keyboard only to copy a file from a mounted NAS storage to ~/Downloads in the files app

### Expected Behavior

Focus highlighting should have more contrast. 
Tab-order should be consistent (top-left to bottom-right, etc.)
Every clickable UI element performing an action when clicked, should also be reachable with the tab key on the keyboard

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

**Comment by jeremypw** [comment]
Yes, keyboard accessibility has rather been neglected :disappointed:  Will have to assign to the ongoing Gtk4 port as Gtk4 handles things differently.

**Comment by CodingSpiderFox** [comment]
At least you have it on the To-Do list, that's great to hear :)

**Comment by jeremypw** [comment]
I am going to convert this to a discussion as there are a number of different issues involved.
